doc/TYPES2021/Demos/HSP.tex

```diff
@@ -1782,30 +1676,28 @@ \subsection{Algebras}\label{algebras}
 
 %% -----------------------------------------------------------------------------
 \paragraph*{Universe levels of algebra types}
-The hierarchy of type universes in \agda is structured as follows:
+Types belong to \emph{universes}, which are structured as follows:
```

Comment:
Actually, Coq and Idris and Lean have universe hierarchies too - only Agda has them so explicit, and non-cumulative. But I'm fine with restoring \agda here.

Reply:
Yes, I know that Coq, Lean, etc have ramified universe hierarchies.  My point---and the reason we need Lift---is that Agda's hierarchy is special *because it's non-cumulative*.  I want to make sure that point is not lost on the reader because this is a major difference that sets Agda apart and, imho, makes Agda a significantly harder language to use.

Reply:
TODO: add "in \agda"  (e.g., "...which, in \agda, are structured as...")